src/typings/schema.graphql.d.ts

```diff
@@ -262,6 +262,8 @@ type DirectoryCtimeArgs = {
 type Site = Node & {
   buildTime: Maybe<Scalars['Date']>;
   siteMetadata: Maybe<SiteSiteMetadata>;
+  port: Maybe<Scalars['Int']>;
```

Comment:
Should this file be here with these changes?

Reply:
nop, for that I opened another PR for it. Can you review it? https://github.com/vtex-sites/base.store/pull/57

This PR will merged after the #57 was merged.